sdk/storage/Azure.Storage.DataMovement.Blobs/stress/src/Azure.Storage.DataMovement.Blobs.Stress.csproj

```diff
@@ -8,16 +8,21 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <!--
+      The test framework pulls in a project reference to Core, which causes a downgrade scenaro against
+      package references. We need an explicit project reference here to hoist the dependency from other
+      packages with transitive dependencies.
+    -->
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Actually I think you can just remove this. I saw that having the "<PackageReference Include="Azure.Core" />" was breaking CI for us, and I just removed it cause it seems to be just fine without it.

Also you may get a merge conflict cause I just merged this PR
https://github.com/Azure/azure-sdk-for-net/pull/54324